addons/hr_timesheet/models/project.py

```diff
@@ -92,9 +93,32 @@ def _compute_remaining_hours(self):
             if group:
                 project.remaining_hours = group.get('remaining_hours')
                 project.has_planned_hours_tasks = bool(group.get('planned_hours'))
+                project.is_project_overtime = group.get('remaining_hours', 0) < 0
             else:
                 project.remaining_hours = 0
                 project.has_planned_hours_tasks = False
+                project.is_project_overtime = False
```

Comment:
We could improve this compute and reduces the number of lines. 🙂 
By doing this:
```suggestion
            project.remaining_hours = group.get('remaining_hours', 0)
            project.has_planned_hours_tasks = bool(group.get('planned_hours'), False)
            project.is_project_overtime = group.get('remaining_hours', 0) < 0
```
What do you think ? 🙂

Reply:
Done!!